Tetris.ino

```diff
@@ -332,7 +349,7 @@ void drawPieceAt(Piece& piece, int x, int y) {
   for(int i = 0; i < piece.width; ++i) {
     for(int j = 0; j < piece.width; ++j) {
       if(piece.shape[i][j] == 1) {
-        arduboy.drawRect(x + j * CELL_SIZE, y + i * CELL_SIZE, CELL_SIZE, CELL_SIZE, WHITE);
+        arduboy.fillRect(x + j * CELL_SIZE, y + i * CELL_SIZE, CELL_SIZE, CELL_SIZE, WHITE);
```

Comment:
I get it. though it does make it easier to look at the next piece and held piece.
this is in prep for ghost piece (a la TGM3)

Once ghost piece works, everything can become a solid.
I intend ghost piece to be a simple black rect (non filled).